### ISSUE TITLE
Write documentation on how to configure the harness for your project

### ISSUE THREAD

**Issue by jameshalsall** [inviqa/harness-go#70]
There are a few mechanisms for extending the harness and a lot of different attributes for configuring it. It would be nice to have some docs available for engineers using this harness so that they have a clear idea of how to work with them. This should be done after #69  and #35 which will change a couple of core extension/customisation points.
